documentation/api/io.strimzi.api.kafka.model.kafka.listener.GenericKafkaListenerConfiguration.adoc

```diff
@@ -27,6 +27,21 @@ listeners:
 
 When the certificate or key in the `brokerCertChainAndKey` secret is updated, the operator automatically detects it in the next reconciliation and triggers a rolling update of the Kafka brokers to reload the certificate.
 
+NOTE: Key referenced in `brokerCertChainAndKey` should be in unencrypted PKCS #8 format. If the certificate is obtained with {CertManagerProject} you can configure {CertManagerPrivateKeyEncoding} to use PKCS #8.
+
+.Example `private key` configured in cert-manager to be formated as unencrypted PKCS #8
+[source,yaml,subs="attributes+"]
+----
+apiVersion: cert-manager.io/v1
+kind: Certificate
+metadata:
+  # ...
+spec:
+  # ...
+  privateKey:
+    encoding: PKCS8
+----
+
```

Comment:
Do we have any other places where we use examples from CertManager? Not sure. I mean, saying it has to be pkcs#8 would be enough for me, then the user has to figure out how (until we have integration with cert-manager within Strimzi).